cmake/LLVMIRUtil.cmake

```diff
@@ -753,14 +753,26 @@ function(llvmir_attach_obj_target)
     list(APPEND IN_FULL_LLVMIR_FILES "${IN_LLVMIR_DIR}/${IN_LLVMIR_FILE}")
   endforeach()
 
-  set(FULL_OUT_LLVMIR_FILE "${WORK_DIR}/${TRGT}.o")
-  if(SHORT_NAME)
```

Comment:
Isn't `SHORT_NAME` still useful for the multiple output files generated by the changes in your PR?

Reply:
In my project it worked fine without it, so I assumed that I could just remove it.

Reply:
Can you please reintroduce it and check that your project still works?
Just because it's not needed in your configuration doesn't mean that wasn't introduced for a reason initially.